src/js/utils.js

```diff
@@ -17,12 +17,28 @@ exports.parseQueryParams = () => {
 /** Creates an object based on a form's current input values. */
 exports.parseForm = form => {
   const formData = new FormData(form);
-  return Array.from(formData.entries()).reduce((acc, [key, value]) => {
-    if (!acc[key]) {
-      acc[key] = [];
+  if (!formData.entries) {
+    // for IE
+    let filter = {};
+    const elements = document.getElementsByTagName("input");
+    for (let i = 0; i < elements.length; i++) {
```

Comment:
Minor, but since there’s a polyfill for `Array.from` now, you could probably do

```
Array.from(elements).forEach((element) => {
  // Do things with element
});
```

Reply:
Yes, that makes it cleaner, done.